autonomi/src/client/data_types/graph.rs

```diff
@@ -42,25 +43,29 @@ pub enum GraphError {
     InvalidQuote,
     #[error("Entry already exists at this address: {0:?}")]
     AlreadyExists(GraphEntryAddress),
+    #[error("Graph forked! Multiple entries found at this address: {0:?}")]
```

Comment:
The `{0:?}` will debug print the `Vec<GraphEntry>`, not only the address.

Reply:
Oh good find!